wasm/src/main/java/com/dylibso/chicory/wasm/types/CustomSection.java

```diff
@@ -10,4 +10,10 @@ protected CustomSection() {
     }
 
     public abstract String name();
+
+    public interface Builder {
+        Builder withBytes(byte[] bytes);
```

Comment:
I think it would be more uniform if every section were parsed the same way. In this case the section builder takes bytes, but in other cases the parsing logic is stored in the `Parser`. What about having a common `Builder` method which parses that section, or making this uniform some other way?